src/Raven.Client/Documents/LinqExtensions.cs

```diff
@@ -1489,6 +1489,17 @@ public static IAsyncDocumentQuery<T> ToAsyncDocumentQuery<T>(this IQueryable<T>
             return ravenQueryInspector.GetAsyncDocumentQuery();
         }
 
+        public static IRavenQueryable<T> Skip<T>(this IQueryable<T> source, long count)
```

Comment:
The idea was to introduce a dedicated method `LongSkip()`. Same as we have `LongCount`:
https://github.com/ravendb/ravendb/blob/97df41d9c9c7a03ba90f443d3ff7d68f6b13a99c/src/Raven.Client/Documents/LinqExtensions.cs#L487
I thought that extension like you added will cause ambiguous reference issues but apparently it doesn't. So maybe that's even better for us.

CC @reebhub @ppekrol